go.mod

```diff
@@ -1,212 +1,107 @@
 module go.mondoo.com/packer-plugin-cnspec
 
-go 1.19
+go 1.21
+
+toolchain go1.21.1
 
 replace github.com/slack-go/slack v0.12.1 => github.com/imilchev/slack v0.0.0-20230324120548-5380d7dd00a5
 
+require go.mondoo.com/cnquery/v9 v9.0.2
```

Comment:
nit: I recommend writing this as:

```
require (
	go.mondoo.com/cnquery/v9 v9.0.2
	go.mondoo.com/cnspec/v9 v9.0.2
)
```

Reply:
@chris-rock I talked with @vjeffrey and asked whether we should bump the go.mod to v9.2.0. Do we not want the latest? 

Also, I updated the example templates and README to reference the latest tag of the plugin.

Reply:
Sure, we can update this. v9.0.2 was the latest version as of the creation of this PR.